src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -3878,7 +3878,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
     <PropertyGroup>
       <_ALExeToolPath>$(TargetFrameworkSDKToolsDirectory)</_ALExeToolPath>
-      <_ALExeToolPath Condition="'$(PlatformTarget)' == 'x64'">$(TargetFrameworkSDKToolsDirectory)$(PlatformTarget)\</_ALExeToolPath>
+      <_ALExeToolPath Condition="'$(PlatformTarget)' == 'x64' and !$(_ALExeToolPath.EndsWith('x64\')">$(TargetFrameworkSDKToolsDirectory)x64\</_ALExeToolPath>
```

Comment:
There is a missing `)`. So, `.EndsWith('x64\')` -> `.EndsWith('x64\'))`

Good news is, this change gets rid of a ton of **MSB3084** warnings I was getting. Here's the warning I was getting in case others report it:

Task attempted to find "al.exe" in two locations. 1) Under the "C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\x64\" processor specific directory which is generated based on SdkToolsPath 2) The x86 specific directory under "C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools\x64\x64\" which is specified by the SDKToolsPath property. You may be able to solve the problem by doing one of the following:  1) Set the "SDKToolsPath" property to the location of the Microsoft Windows SDK.

Reply:
ah good catch!

Do you happen to know where it had `x64\` appended prior to this target running? The original issue suggested it was never getting appended, but it sounds like other packages were manually appending it and now this is conflicting.

Reply:
> Do you happen to know where it had x64\ appended prior to this target running? The original issue suggested it was never getting appended, but it sounds like other packages were manually appending it and now this is conflicting.

Oh, geez...It looks like someone implemented a suggestion from [stackoverflow](https://stackoverflow.com/questions/25365616/alink-warning-al1073-referenced-assembly-mscorlib-dll-targets-a-different-p/41945190#41945190)  in our codebase to alleviate `AL1073` warnings. So your fix in #6484 means that can be removed. Kindly disregard 😊.